src/components/images/imageLoader.js

```diff
@@ -12,7 +12,7 @@ import 'css!./style';
         fillImageElement(elem, source);
     }
 
-    async function itemBlurhashing(target, blurhashstr) {
```

Comment:
Already asked you in Matrix, but I'll do it here again so it can be discussed publicly.

Why?

Reply:
It's useless in this case. Quoting MDN:

> The async and await keywords enable asynchronous, promise-based behavior to be written in a cleaner style, avoiding the need to explicitly configure promise chains.

This doesn't make use of Promises so it is not needed here (But I do want to clean up all the Promise.then() everywhere with async/await, at some point).